subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/espresso/action/click/ClickAction.kt

```diff
@@ -61,16 +62,19 @@ class ClickAction(
                 coordinates: FloatArray,
                 precision: FloatArray
             ) {
+                val clickVisualization = ClickVisualization(coordinates[0], coordinates[1])
```

Comment:
This feature has to be optional for performance tests

Reply:
Fixed in 85d51aa32dabd529c5c6c9ec2bde91d27ac58c22